spanner-2019-10-01.ddl

```diff
@@ -37,7 +37,7 @@ CREATE TABLE bsos (
 INTERLEAVE IN user_collections;
 
     CREATE INDEX BsoExpiry
-        ON bsos(expiry),
+        ON bsos(fxa_uid, fxa_kid, collection_id, expiry),
```

Comment:
Would it make sense to make a new DDL file with a new date instead of changing the old one? (Just so we have a historical record of changes?)

Reply:
I've been reluctant to do something like this (or applying changes as DDL modifications) until #286 is implemented. It might look similar to our [mysql backend migrations structure](https://github.com/mozilla-services/syncstorage-rs/tree/master/migrations) but that's not 100% clear it's implemented (and we always have git history? :sob:)

#286 should become higher priority once we're closer to a steadier prod and other even higher priority bits are finished.